packages/neuron-wallet/src/block-sync-renderer/sync/indexer-cache-service.ts

```diff
@@ -92,12 +92,12 @@ export default class IndexerCacheService {
           this.indexer,
           {
             lock: {
-              code_hash: lockScript.codeHash,
-              hash_type: lockScript.hashType,
+              codeHash: lockScript.codeHash,
+              hashType: lockScript.hashType,
               args: lockScript.args,
-            },
+            }
           },
-          this.indexer.ckbRpcUrl,
+          this.indexer.ckbRpcUrl!,
```

Comment:
I think it's better to return when `this.indexer.ckbRpcUrl` is undefined. Because when using `!`, we cannot predict what happened